virtio-devices/src/balloon.rs

```diff
@@ -318,8 +321,16 @@ pub struct Balloon {
 
 impl Balloon {
     // Create a new virtio-balloon.
-    pub fn new(id: String, size: u64, seccomp_action: SeccompAction) -> io::Result<Self> {
-        let avail_features = 1u64 << VIRTIO_F_VERSION_1;
+    pub fn new(
+        id: String,
+        size: u64,
+        deflate_on_oom: bool,
```

Comment:
Since you now use two fields from `BalloonConfig`, I think you can just pass in that structure here. This reduces code churn in the future when someone implements more features that are supported by VirtIO balloon.

Reply:
> Since you now use two fields from `BalloonConfig`, I think you can just pass in that structure here. This reduces code churn in the future when someone implements more features that are supported by VirtIO balloon.

Good point. But I notice the BalloonConfig struct is defined in vmm/src/config.rs, while vmm package depends on virtio-devices package. Trying to replace the two fields with BalloonConfig struct means we need use vmm/* inside virtio-device/*, which will cause the cyclic error I am afraid. :(

Reply:
Fair enough. I didn't notice that.